### PR TITLE
Completed Basic Functionality Tasks

### PR DESCRIPTION
**Pull request by juliareichel**
This includes updating the README.md, adding functionality, and styling

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,11 +8,84 @@
...
+            <li> I am currently the Executive Vice President and in the process of co-founding a chapter of The Women's Network, a national women-led networking organization 
+                focused on connecting ambitious undergraduate students with professional workers.</li> 
+         </ol> 
+    
+      <p>During my free time in quarantine I: <p>
+
+           Watched shows on Netflix and HBO:
+          <ul class="shows">
+              <li> Outer Banks </li>
+              <li> Big Little Lies </li>
```
Comment (damargulis) [inline portfolio/src/main/webapp/index.html:31]: My sister loves this show! Was thinking of watching soon, was it good?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -13,16 +13,17 @@
...
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
```
Comment (damargulis) [inline portfolio/src/main/webapp/script.js:19]: We usually make constants top-level, and use UPPER_CASE_LETTERING

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (juliareichel) [reply portfolio/src/main/webapp/index.html:31]: Yes I loved it! Read the book first

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -8,11 +8,84 @@
...
+              <li> Peloton live streams</li>
+          </ul>
+      <img id="portrait" src="/images/myself.jpg">
+      <button id="fact_btn" onclick="addRandomFact()">Click to see a fun fact!</button>
+      <div id="fact-container"></div>
+      <p id="header"> Dream Vacation Spots </p>
+      <div class="vacation_spots">
+          <div class = "destination">
+            <img src="/images/capri.jpeg"> <br>
+            <span class="place">Capri</span>
```
[the remark this answers is not in the document]
Comment (juliareichel) [reply portfolio/src/main/webapp/index.html:58]: yes, I fixed it in a more recent version

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -8,11 +8,84 @@
...
+          </div>
+          <div class = "destination">
+            <img src="/images/costarica.jpeg"> <br>
+            <span class="name">Costa Rica</span>
+          </div>
+          <div class = "destination">
+            <img src="/images/mykonos.jpeg"> <br>
+            <span class="name">Mykonos</span>
```
[the remark this answers is not in the document]
Comment (juliareichel) [reply portfolio/src/main/webapp/index.html:66]: would love to see pictures!

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (juliareichel) [reply portfolio/src/main/webapp/script.js:19]: by "top-level" do you mean outside the function declaration?

Comment (damargulis) [reply portfolio/src/main/webapp/script.js:19]: Yep exactly! So should look something like
```
const FACTS = [
    'fact1',
    'fact2',
];

function() {
}
```

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -8,11 +8,84 @@
...
+          <ul>
+              <li> Chloe Ting Summer Shred </li>
+              <li> Alexis Ren workouts </li>
+              <li> 305 Dance recordings</li>
+              <li> Peloton live streams</li>
+          </ul>
+      <img id="portrait" src="/images/myself.jpg">
+      <button id="fact_btn" onclick="addRandomFact()">Click to see a fun fact!</button>
+      <div id="fact-container"></div>
+      <p id="header"> Dream Vacation Spots </p>
```
[the remark this answers is not in the document]
Comment (juliareichel) [reply portfolio/src/main/webapp/index.html:54]: also changed this in updated version

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (juliareichel) [reply portfolio/src/main/webapp/script.js:19]: perfect!

File: portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -4,6 +4,27 @@
...
+#fact_btn {
+    padding: 10px; 
+    border-radius: 5%;
+}
+
+#header {
+    text-align: center
+}
+
+.fact_styling {
```
Comment (ryanthompson591) [inline portfolio/src/main/webapp/style.css:21]: calling a styling styling is redundant.  In general try to avoid redundant naming,

Comment (juliareichel) [reply portfolio/src/main/webapp/style.css:21]: fixed

**Pull request merged**